index.bs

```diff
@@ -696,10 +696,16 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     1. If <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> [=list/is not empty=], execute a 
         platform-specific procedure to determine which, if any, credentials in
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> are present on this |authenticator| by matching with 
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}.{{ScopedCredentialDescriptor/id}}</code> and 
-        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}.{{ScopedCredentialDescriptor/type}}</code>, and set |credentialList| to
-        this filtered list.
+        <code>|scopedOptions|.{{ScopedCredentialRequestOptions/allowList}}</code> are present on this |authenticator| by
```

Comment:
This is a big step. ;) I might split it up something like:

[Remove "Let credentialList be a new list."]

```html
1. If scopedOptions.allowList:
<dl class="switch">
    : [=list/is empty=]
    :: Let |credentialList| be a new [=list=] consisting of the
        credentials on |authenticator| that are mapped to
        rpId and also have a type within scopedOptions.allowList.
        [Or you could use "match" from below.]
        <span class="note">Note: the procedure to identify these
        credentials is platform-specific.</span>
    : [=list/is not empty=]
    :: Let |credentialList| be a new [=list=] consisting of the
        credentials on |authenticator| that match [Ideally, "match"
        would be a link to a definition of how a credential matches
        a ScopedCredentialDescriptor.] an item in
        scopedOptions.allowList. <span class="note">Note: the
        procedure to identify these credentials is
        platform-specific.</span>
```